docs/index.html.ejs

```diff
@@ -720,6 +720,83 @@
       </div>
     </section>
 
+    <section class="component">
+      <h3 id="tabs">Tabs</h3>
+      <div>
+        <blockquote>
+          ...
```

Comment:
Looks like the quote's missing

Reply:
Also, the tabs header isn't in the sidebar.
Any ETA for when this will be merged?